.changeset/start_clijs_deprecation.md

```diff
@@ -0,0 +1,5 @@
+---
+"@asyncapi/generator": minor
+---
+
+Start deprecating `cli.js` in the generator repository.
```

Comment:
what do you think about providing a bit more context here. This description is later visible under a release name  and I can guarantee you that many people look into GitHub Release sections to understand quickly what is the goal of given release.

example description of last release: https://github.com/asyncapi/generator/releases/tag/%40asyncapi%2Fgenerator%402.4.0

so what about giving more context and linking to doc with more inf?